eng/pipelines/coreclr/templates/run-superpmi-collect-job.yml

```
@@ -100,6 +100,12 @@ jobs:
     - ${{ if eq(parameters.collectionName, 'libraries_tests') }}:
       - name: InputDirectory
         value: '$(Build.SourcesDirectory)/artifacts/tests/libraries/$(osGroup).$(archType).$(buildConfigUpper)'
+    - ${{ if and(eq(parameters.collectionName, 'smoke_tests'), eq(parameters.osGroup, 'windows')) }}:
+      - name: InputDirectory
+        value: '$(Build.SourcesDirectory)\artifacts\tests\coreclr\obj\${{ parameters.osGroup }}.${{ parameters.archType }}.${{ parameters.buildConfig }}\Managed\nativeaot\SmokeTests'
+    - ${{ if and(eq(parameters.collectionName, 'smoke_tests'), ne(parameters.osGroup, 'windows')) }}:
+      - name: InputDirectory
+        value: '$(Build.SourcesDirectory)/artifacts/tests/coreclr/obj/${{ parameters.osGroup }}.${{ parameters.archType }}.$(buildConfigUpper)/Managed/nativeaot/SmokeTests'
```

The `libraries_tests` case above doesn't require windows and non-windows cases. So I would expect this doesn't, either.
```suggestion
    - ${{ if eq(parameters.collectionName, 'smoke_tests')) }}:
      - name: InputDirectory
        value: '$(Build.SourcesDirectory)/artifacts/tests/coreclr/obj/$(osGroup).$(archType).$(buildConfigUpper)/Managed/nativeaot/SmokeTests'
```